tasks/client_config.yaml

```
@@ -1,10 +1,33 @@
 ---
+- name: Check for root password
+  changed_when: false
+  stat:
+    path: /usr/local/etc/mysql-password
+  register: mariadb_mysql_password
+
+- name: create root password
+  when: mariadb_mysql_password.stat.exists == false
+  copy:
+    dest: /usr/local/etc/mysql-password
+    content: "{{ lookup('password', '/dev/null length=32 chars=ascii_letters,digits') }}"
+    owner: "{{ system.user.root.name }}"
+    group: "{{ system.user.root.group }}"
```

Has been moved to the `mariadb` namespace.
```suggestion
    owner: "{{ mariadb.system.user.root.name }}"
    group: "{{ mariadb.system.user.root.group }}"
```